tor/core/user_interaction.py

```diff
@@ -35,6 +35,24 @@ def coc_accepted(post, config):
     return config.redis.sismember('accepted_CoC', post.author.name) == 1
 
 
+user_commands = {}
```

Comment:
I have to get to my desktop to test this, but as I'm reading this is seems like we're importing this dict and never filling it. Am I reading this wrong?

Reply:
Thanks, I've never seen it done that way and I missed that. I've tested it manually and this is probably one of the cleanest changes we've ever had to the codebase. :tada: :tada:

Reply:
As @itsthejoker mentioned earlier in this thread, using decorators adds a level of indirection to the code that would make tracing it a bit more difficult. That said this is a lot clearer in what we're trying to do overall by hiding the implementation details. I like it. 👍